modules/installation-azure-finalizing-encryption.adoc

```diff
@@ -0,0 +1,83 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-customizations.adoc
+// * installing/installing_azure/installing-azure-government-region.adoc
+// * installing/installing_azure/installing-azure-network-customizations.adoc
+// * installing/installing_azure/installing-azure-private.adoc
+// * installing/installing_azure/installing-azure-vnet.adoc
+
+:_content-type: PROCEDURE
+[id="finalizing-encryption_{context}"]
+= Finalizing user-managed encryption after installation
+If you installed {product-title} using a user-managed encryption key, you can complete the installation by creating a new storage class and granting write permissions to the Azure cluster resource group.
+
+.Procedure
+. Obtain the identity of the cluster resource group used by the installer:
+.. If you specified an existing resource group in `install-config.yaml`, obtain its Azure identity:
++
+[source,terminal]
+----
+$ az identity list --resource-group "<existing_resource_group>"
+----
++
+.. If you did not specify a existing resource group in `install-config.yaml`, locate the resource group that the installer created, then obtain its Azure identity:
++
+[source,terminal]
+----
+$ az group list
+----
++
+[source,terminal]
+----
+$ az identity list --resource-group "<installer_created_resource_group>"
+----
+. Grant a role assignment to the cluster resource group so that it can write to the Disk Encryption Set:
++
+[source,terminal]
+----
+$ az role assignment create --role "<privileged_role>" --assignee "<resource_group_identity>"
+----
++
+where:
+
+`<privileged_role>`:: Specifies an Azure role that has read/write permissions to the disk encryption set. You can use the `Owner` role or a custom role with the necessary permissions.
+`<resource_group_identity>`:: Specifies the identity of the cluster resource group.
++
+. Create a storage class that uses the user-managed disk encryption set:
+.. Save the following storage class definition to a file, for example `storage-class-definition.yaml`:
++
+[source,terminal]
+----
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: managed-premium
+provisioner: kubernetes.io/azure-disk
+parameters:
+  skuname: Premium_LRS
+  kind: Managed
+  diskEncryptionSetID: "<disk_encryption_set_ID>"
+  resourceGroup: <resource_group_name>
+reclaimPolicy: Delete
+allowVolumeExpansion: true
+volumeBindingMode: WaitForFirstConsumer
+----
++
+where:
```

Comment:
I recommend using this method for user readability:
```
----
kind: StorageClass
apiVersion: storage.k8s.io/v1
metadata:
  name: managed-premium
provisioner: kubernetes.io/azure-disk
parameters:
  skuname: Premium_LRS
  kind: Managed
  diskEncryptionSetID: "<disk_encryption_set_ID>" <1>
  resourceGroup: <resource_group_name> <2>
reclaimPolicy: Delete
allowVolumeExpansion: true
volumeBindingMode: WaitForFirstConsumer
----
+
<1> Specifies the ID of the disk encryption  ...
<2> Specifies the name .... 
```
Please reach out for specific examples in our documentation if you like!